src/components/ChatEntry.js

```diff
@@ -1,22 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onUpdateLike }) => {
+  const [heart, setHeart] = useState(liked ? '❤️' : '🤍');
```

Comment:
This works on the first render of the component, displaying the empty heart emoji. The problem is that no functions then update the state when the heart emoji is clicked. 

So here are two options:
1.  Create a function that will a) update `heart` state _and_ also b) call the `onUpdateLike` function.
2.  Remove the `heart` state altogether, and use the `liked` from props instead. Then move your ternary expression on line 7 `liked ? '❤️' : '🤍'` to line 17 like this: `{liked ? '❤️' : '🤍'}`